src/ui/components/Account/index.js

```diff
@@ -56,6 +77,20 @@ function FirstRecordingPrompt() {
   );
 }
 
+function getRecordings(data) {
+  if (!data?.users.length) {
```

Comment:
Do we need the optional chaining here? We check for `error`/`loading` in the AccountPage component, so by the time we pass the data here it _should_ be truthy.

Reply:
I guess not...